testing/bdd/features/styles/ebuttd_fontsize_same_style_ref.feature

```diff
@@ -0,0 +1,60 @@
+  | 32 15           |              | 1c       | normal    |                  |                    |                |                  |
+  | 32 15           |              | 2c       | normal    | 200%             |                    |                |                  |
+  | 32 15           |              | 100%     | normal    |                  |                    |                |                  |
+  | 32 15           |              | 200%     | normal    | 200%             |                    | 200%           |                  |
+  | 32 15           |              |          | normal    |                  |                    |                |                  |
+  | 32 15           | 1280px 720px | 24px     | normal    | 50%              |                    |                |                  |
+  | 32 15           | 1280px 720px | 48px     | normal    |                  |                    |                |                  |
+  | 40 24           |              | 1c 2c    | 120%      | 200%             | 120%               |                |                  |
+  | 32 15           |              | 1c       | 120%      |                  | 120%               |                |                  |
+  | 32 15           |              | 2c       | 120%      | 200%             | 120%               |                |                  |
+  | 32 15           |              | 100%     | 120%      |                  | 120%               |                |                  |
+  | 32 15           |              | 200%     | 120%      | 200%             | 120%               | 200%           | 120%             |
+  | 32 15           |              |          | 120%      |                  | 120%               |                |                  |
+  | 32 15           | 1280px 720px | 24px     | 120%      | 50%              | 120%               |                |                  |
+  | 32 15           | 1280px 720px | 48px     | 120%      |                  | 120%               |                |                  |
+  | 40 24           |              | 1c 2c    | 2.4c      | 200%             | 120%               |                |                  |
+  | 32 15           |              | 1c       | 1.2c      |                  | 120%               |                |                  |
+  | 32 15           |              | 2c       | 2.4c      | 200%             | 120%               |                |                  |
+  | 32 15           |              |          | 120%      |                  | 120%               |                |                  |
+  | 32 15           | 1280px 720px | 24px     | 28.8px    | 50%              | 120%               |                |                  |
+  | 32 15           | 1280px 720px | 48px     | 57.6px    |                  | 120%               |                |                  |
```

Comment:
Would it be possible to have a couple of different values for the calculated line height? Could a couple of those be changed to 150% instead, just so there's a variety. It might be easier to work out how that value gets calculated.

Reply:
Fixed in f1201f6